### ISSUE TITLE
Hosted version of safe-settings

### ISSUE THREAD

**Issue by LamarrD** [github/safe-settings#260]
## Prerequisites:

* Is the functionality available in the GitHub UI? If so, please provide a link to information about the feature.
N/A

* Is the functionality available through the GitHub API? If the functionality is available, please provide links to the
  API documentation (https://developer.github.com/v3/) as well as the Octokit documentation (https://octokit.github.io/).
N/A

* If the functionality is not yet available in the API, it would be helpful if you
  contacted support (https://support.github.com/) or posted in the Community Forum (https://github.community/). Please
  include a link to the forum post if you create one or a copy of the response from support.
N/A

## New Feature

Please describe the desired new functionality:
This is more of a question than a feature request. The documentation says "See [docs/deploy.md](https://github.com/github/safe-settings/blob/main-enterprise/docs/deploy.md) if you would like to run your own instance of this plugin." Does that mean not running our own instance of the plugin is an option?

In other words, is there a hosted option for safe-settings from GitHub?

**Comment by decyjphr** [comment]
Hi @LamarrD There is a hosted version of the app. It is a public GH App. However, it is only suited for dev/test. We have not dedicated resources towards making it a production-grade version because there isn't a lot of demand for a public version. If you are interested in testing it for Dev/Test, you can go to this URL https://github.com/apps/safe-settings-app

**Comment by LamarrD** [comment]
Thanks @decyjphr! Out of curiosity what makes it only suitable for Dev/Test. Is it just due to not having support, scalability, or security?

**Comment by decyjphr** [comment]
It is mostly because of the lack of support, scalability, and availability.  There are security implications when installing public GH Apps because you would be trusting an external app with admin permissions to your Org, and that could be problematic depending on your industry, and compliance requirements.

**Comment by kiview** [comment]
> because there isn't a lot of demand for a public version

As a maintainer of an OSS organization, I would see demand in this area (since we have no interest in hosting and operating our own services). Is the public version just hosted and maintained by you @decyjphr, or is there any involvement from GH?

**Comment by decyjphr** [comment]
Only by me (I pay for it but keep a cap on it)
